app/views/authentication_mailer/sign_in_email.text.erb

```diff
@@ -1,5 +1,5 @@
 # <%= t('authentication.sign_in.email.subject') %>
 
-Click the link below to sign in to the Apply for teacher training service.
+Click the link below to sign in to the <%= t('service_name.apply') %> service.
 
```

Comment:
Emails don’t have access to `service_name` variable, so we call translated string directly.